client/boot/polyfills.js

```diff
@@ -1,7 +0,0 @@
-import '@automattic/calypso-polyfills';
```

Comment:
Did we actually mean to remove this one? Doesn't this still include some `core-js` polyfills?

I'd expect if we intend to remove this one, we'd do it separately so we can test and ensure nothing is broken, etc.

Reply:
> Did we actually mean to remove this one? Doesn't this still include some `core-js` polyfills?

Restored in d27c6bde06, though moved it to be imported directly from `client/boot/app.js` instead of keeping this file as a pass-through.